src/core/src/sim/mc/moves/resize.rs

```diff
@@ -81,13 +81,13 @@ impl MCMove for Resize {
         };
 
         for (mi, molecule) in self.previous.molecules().iter().enumerate() {
-            // We don't want to change the intramolecular distances
-            // so we compute the translation vector of the center-of-mass
-            // (com) of a molecule and apply it to all its particles.
-            // Note that to do this, the com of a molecule *always* has
-            // to reside inside the simulation cell.
+            // We don't want to change the intramolecular distances so we
+            // compute the translation vector of the center-of-mass (com) of a
+            // molecule and apply it to all its particles. Note that to do
+            // this, the com of a molecule *always* has to reside inside the
+            // simulation cell.
             let old_com = system.molecule_com(mi);
```

Comment:
It doesn't make a difference, but I think it would be more clear to use
```rust
let old_com = self.previous.molecule_com(mi);
```
here. Just a nitpick, we can also leave it as is.

Reply:
Yes, this seems more clear =)